analyzer/quality-analyzer/src/main/java/eu/fasten/analyzer/qualityanalyzer/QualityAnalyzerPlugin.java

```diff
@@ -105,6 +110,13 @@ public void consume(String kafkaMessage) {
                 catch(DataAccessException e) {
                     logger.info("Data access exception");
                     // Database connection error
+                    // The error codes starting with 57P0 are related to the DB connection issues.
+                    // See https://www.postgresql.org/docs/current/errcodes-appendix.html
+                    if (e.sqlState().contains("57P0")) {
```

Comment:
See previous discussion.

Reply:
Addressed in [b9dbc27](https://github.com/fasten-project/fasten/pull/466/commits/b9dbc27c6b91895e7a28b43d985207bf883e22b7).